### PR TITLE
Fix conflicting alchemy recipe for Prismarine Shard and Silicone Caulk Bucket

### PR DESCRIPTION
**Pull request by akisephila**
Change Silicone Caulk Bucket recipe from Water Bucket && Quartz to Water Bucket && Amethyst Shard

Remove unused import statements from MinestuckCombinationsProvider

### CODE REVIEW

**Comment by kirderf1** [comment]
Just need to merge 1.20.1 and add a changelog entry as far as I'm concerned.

**Comment by akisephila** [comment]
> Just need to merge 1.20.1 and add a changelog entry as far as I'm concerned.

Updated.

File: CHANGELOG.md
Comment (kirderf1) [inline CHANGELOG.md:1]: This change is added under an already released version. It should be added under the unreleased section.

Also I recommend writing the recipe change and conflict fix as one change.

Comment (akisephila) [reply CHANGELOG.md:1]: Fixed

Comment (kirderf1) [reply CHANGELOG.md:1]: And add your name as a contributor in unreleased.

Comment (akisephila) [reply CHANGELOG.md:1]: Fixed

**Review by kirderf1 — APPROVED** [review]

**Pull request merged**